<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PicsouDBViewer</class>
 <widget class="QWidget" name="PicsouDBViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>717</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="db_info_group">
     <property name="title">
      <string>DB information</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="version_label">
            <property name="text">
             <string>Version:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="name_label">
            <property name="text">
             <string>Name:</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="version">
            <property name="text">
             <string>&lt;version&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="name">
            <property name="text">
             <string>&lt;name&gt;</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="description_label">
        <property name="text">
         <string>Description:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="description">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::TextBrowserInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="users_group">
     <property name="title">
      <string>Users</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QListWidget" name="users_list">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="showDropIndicator" stdset="0">
         <bool>false</bool>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="textElideMode">
         <enum>Qt::ElideNone</enum>
        </property>
        <property name="sortingEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="users_buttons">
        <item>
         <widget class="QPushButton" name="add_user">
          <property name="text">
           <string> Add</string>
          </property>
          <property name="icon">
           <iconset resource="../../picsou.qrc">
            <normaloff>:/resources/material-design/svg/account-plus.svg</normaloff>:/resources/material-design/svg/account-plus.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="edit_user">
          <property name="text">
           <string> Edit</string>
          </property>
          <property name="icon">
           <iconset resource="../../picsou.qrc">
            <normaloff>:/resources/material-design/svg/account-edit.svg</normaloff>:/resources/material-design/svg/account-edit.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="remove_user">
          <property name="text">
           <string> Remove</string>
          </property>
          <property name="icon">
           <iconset resource="../../picsou.qrc">
            <normaloff>:/resources/material-design/svg/account-remove.svg</normaloff>:/resources/material-design/svg/account-remove.svg</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="action_add_user">
   <property name="icon">
    <iconset resource="../../picsou.qrc">
     <normaloff>:/resources/material-design/svg/account-plus.svg</normaloff>:/resources/material-design/svg/account-plus.svg</iconset>
   </property>
   <property name="text">
    <string>Add user</string>
   </property>
  </action>
  <action name="action_edit_user">
   <property name="icon">
    <iconset resource="../../picsou.qrc">
     <normaloff>:/resources/material-design/svg/account-edit.svg</normaloff>:/resources/material-design/svg/account-edit.svg</iconset>
   </property>
   <property name="text">
    <string>Edit user</string>
   </property>
  </action>
  <action name="action_remove_user">
   <property name="icon">
    <iconset resource="../../picsou.qrc">
     <normaloff>:/resources/material-design/svg/account-remove.svg</normaloff>:/resources/material-design/svg/account-remove.svg</iconset>
   </property>
   <property name="text">
    <string>Remove user</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>users_list</tabstop>
  <tabstop>add_user</tabstop>
  <tabstop>edit_user</tabstop>
  <tabstop>remove_user</tabstop>
  <tabstop>description</tabstop>
 </tabstops>
 <resources>
  <include location="../../picsou.qrc"/>
 </resources>
 <connections/>
</ui>
